src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,42 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int PASSWORD_MIN_LENGTH = 6;
+    private static final int AGE_MIN = 18;
+    private static final int AGE_MAX = 122;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
-    public User register(User user) {
-        return null;
+    public User register(User user) throws RegistrationServiceException {
+        if (user.getLogin() == null) {
+            throw new RegistrationServiceException("Error, login is " + user.getLogin());
+        }
+        if (user.getLogin().equals("")) {
+            throw new RegistrationServiceException("Error, login is " + user.getLogin());
```

Comment:
It will be hard to see empty string in error message obtained from .getLogin() method
```suggestion
            throw new RegistrationServiceException("Error, login can't be empty");
```